cmd/logger/audit.go

```diff
@@ -27,23 +30,43 @@ import (
 // ResponseWriter - is a wrapper to trap the http response status code.
 type ResponseWriter struct {
 	http.ResponseWriter
-	statusCode      int
+	StatusCode      int
 	startTime       time.Time
 	timeToFirstByte time.Duration
+	// number of bytes written
+	bytesWritten int
+	// Response body should be logged
+	LogBody bool
```

Comment:
can all the exposed fields come together and all the private after for readability?